target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -55,6 +55,11 @@ ipq40xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0u@eth0" "3:lan" "4:lan"
 		;;
+	mobipromo,cm520-79f)
+		ucidef_add_switch "switch0" \
+			"0u@eth0" "3:lan:2" "4:lan:1"
+		ucidef_set_interface_wan "eth1"
```

Comment:
Typically, ucidef_set_interface_wan is put before ucidef_add_switch. But that's a cosmetical issue.

Reply:
I put `ucidef_set_interface_wan` after `ucidef_add_switch` so that interface wan is added after lan.